sdks/python/apache_beam/examples/cookbook/mergecontacts_test.py

```diff
@@ -18,10 +18,10 @@
 """Test for the mergecontacts example."""
 
 import logging
-import tempfile
+import re
 import unittest
 
-from apache_beam.examples.cookbook import mergecontacts
```

Comment:
(Similarly for most of the rest of the cookbook examples, e.g. group_with_coder_test. If there are non-trivial transforms, we could add tests for those in isolation.)

Changing the tests of functionality that has nothing to do with IO, however, is very useful.